### ISSUE TITLE
Unable to find AD game

### ISSUE THREAD

**Issue by girdfoxgig** [ValveSoftware/Dota2-Gameplay#23363]
### Description

I have not been able to queue into a single AD match. I've tried about 5 times only for it to get to 45 minutes and I give up. High behaviour, bunch of regions, tried queuing up at peak hours, but nothing.
Thanks

**Comment by ogresleftheadisgay** [comment]
Yeah no one plays it thats why...  might be useful to know your score and report rating to figure out if you are just shadowbanned https://steamcommunity.com/my/gcpd/570/?category=Account&tab=MatchPlayerReportIncoming, take a screenshot of this, make sure it includes your BS on the right hand side

**Comment by girdfoxgig** [comment]
Hey there,

I used to able to find a game around 10ish mins and my friend can still
find a game relatively easily. I think its just me having issues.
i have BS over 10K.

Thanks

[image: image.png]

**Comment by ogresleftheadisgay** [comment]
Shadowbanned. Youll never receive a response in here

**Comment by girdfoxgig** [comment]
Sorry what’s shadow banned? All the bestEthan YuOn Dec 22, 2024, at 9:19 PM, GeeTookYouLongEnough ***@***.***> wrote:﻿
Shadowbanned. Youll never receive a response in here
